openfisca-web-api/deprecated-api.md

```diff
@@ -0,0 +1,639 @@
+# Deprecated API
+
+> This is the documentation for the old OpenFisca web API.  
+> Note that it isn't updated and that its stability isn't guaranteed anymore.  
+> Please use our [last API](http://openfisca.org/doc/openfisca-web-api) instead.  
+
+## Public instance for France
+
+The OpenFisca project still provides an instance of the deprecated API, complete with the French tax and benefit system, on [`api.openfisca.fr`](https://api.openfisca.fr). But we recommand you an instance of the last API serving the French tax and benefit system.  
+For more information, see this documentation on the [public France API](http://openfisca.org/doc/openfisca-web-api#public-france-api).
```

Comment:
This link is not the right one for the job :)

Reply:
"For more information" is about the new API. 😳 Updated.